platform/src/Playground.js

```diff
@@ -867,6 +900,39 @@ function savePanelContents(event){
     });
 }
 
+/**
+ * Poll for editor to become available. 
+ * @param {*} statusUrl
+ * @param {*} url 
+ * @param {*} editorPanel 
+ * @param {*} editorActivity TODO remove as this can be found to save having to specify in config.
+ */
+async function checkEditorReady(statusUrl, url, editorPanel, editorActivity){
+
+   let response  = await fetch(statusUrl);
+
+   if (response.status == 200){ 
+        const result = await response.json();
+
+        if (!result.editorReady){
+            await new Promise(resolve => setTimeout(resolve, 2000));
+            await checkEditorReady(statusUrl, url, editorPanel, editorActivity);
```

Comment:
Why are we doing this active waiting? Shouldn't this simply be an HTTP call that waits for a promise to get the result?

Reply:
Agree that we should use server sent events to detect this rather than polling, added task to #81 .